src/app/commands/cmd_options.cpp

```diff
@@ -1756,6 +1760,17 @@ class OptionsWindow : public app::gen::Options {
     return paths;
   }
 
+  static base::paths createUserDirPaths(const base::paths dirNames) {
```

Comment:
I'd change the name of this function to something like
```suggestion
  static base::paths getUserDirPaths(const base::paths dirNames) {
```
because when I saw it for first time I thought it was creating (mkdir) the directories.